### PR TITLE
Adding publish iis as a project and minor cleanup

### PR DESCRIPTION
**Pull request by JunTaoLuo**
@moozzyk I think you missed a few things.

### CODE REVIEW

**Comment by moozzyk** [comment]
I did the change in VS Code. I have the same changes in the pipeline.

File: IISIntegration.sln [hunk 1]
```diff
@@ -25,6 +25,8 @@ Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "samples", "samples", "{C74B
 EndProject
 Project("{8BB2217D-0F2D-49D1-97BC-3654ED321F3B}") = "IISSample", "samples\IISSample\IISSample.xproj", "{E4E2BDC4-A9C6-4AE9-B429-032EC83EDE64}"
 EndProject
+Project("{8BB2217D-0F2D-49D1-97BC-3654ED321F3B}") = "Microsoft.AspNet.Tools.PublishIIS", "src\Microsoft.AspNet.Tools.PublishIIS\Microsoft.AspNet.Tools.PublishIIS.xproj", "{D459A568-7404-43FA-91B0-D7F5D5EC6D14}"
```
Comment (moozzyk) [inline IISIntegration.sln]: test project is missing..

File: src/Microsoft.AspNet.Tools.PublishIIS/Program.cs [hunk 2]
```diff
@@ -63,7 +66,7 @@ public static int Main(string[] args)
 #if DEBUG
                 Console.Error.WriteLine(e);
 #else
-                Console.Error.WriteLine(ex.Message);
+                Console.Error.WriteLine(e.Message);
```
Comment (moozzyk) [inline src/Microsoft.AspNet.Tools.PublishIIS/Program.cs]: this is a very good catch - I ran only debug...

File: src/Microsoft.AspNet.Tools.PublishIIS/WebConfigTransform.cs [hunk 3]
```diff
@@ -1,3 +1,6 @@
+// Copyright (c) .NET Foundation. All rights reserved.
```
Comment (moozzyk) [inline src/Microsoft.AspNet.Tools.PublishIIS/WebConfigTransform.cs]: You are now officially the "@troydai missing header" person :)

**Comment by JunTaoLuo** [comment]
:up::date:

**Comment by moozzyk** [comment]
:ship::it:

(Thanks for fixing this for me :bow:)

**Comment by JunTaoLuo** [comment]
no problem!

**Pull request merged**